org.eclipse.draw2d.tests/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.draw2d.tests
-Bundle-Version: 3.12.200.qualifier
+Bundle-Version: 3.12.300.qualifier

```

Comment:
I already increased the maintenance version number when doing the 3.19 preparations. Is this then still needed?

As we have added the new factory in draw2d and the tests using it should we change the version number to 3.15 to show that the tests are aligned? Or at least change the minor version number?

Reply:
> As we have added the new factory in draw2d and the tests using it should we change the version number to 3.15 to show that the tests are aligned? Or at least change the minor version number?

I don't think we should jump from 3.12.200 to 3.15, but at the very least to 3.13, because new tests have been "added" (i.e. reactivated).

Reply:
This sounds like a plan.